pkg/kubectl/cmd/util/helpers.go

```diff
@@ -64,6 +64,10 @@ func AddSourceToErr(verb string, source string, err error) error {
 
 var fatalErrHandler = fatal
 
+// ApplyAnnotation is the annotation used to store the previous configuration
+// of a resource that can be used in a three way diff by UpdateApplyAnnotation.
+const ApplyAnnotation = "k8s-kubectl-apply"
```

Comment:
I looked for precedent, and found only the labels used by add-ons, which I emulated. Do we have examples of annotations or labels which use the pattern you're recommending? If not, why is it better than the pattern used by add-on labels?